### DISCUSSION TITLE
Welcome to DDB Gamelog discussions!

### DISCUSSION THREAD

**Discussion by IamWarHead** [IamWarHead/ddb-game-log#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by MaineKnight** [comment]
Question:

Is this module designed for a single user? I ask, because our in person game uses dnd beyond for all players but I use foundry for everything else. Can I set up foundry to work if I have 5 player tokens or does the module assume 1 token per player on the laptop?
